src/inc/bbsweep.h

```diff
@@ -196,7 +196,7 @@ class BBSweep
             if (!OpenProcessToken(GetCurrentProcess(), TOKEN_QUERY, &hToken))
                 goto cleanup;
 
-            // don't set pSecurityAttributes for Metro processes
+            // don't set pSecurityAttributes for Windows Store processes
```

Comment:
This should all talk about AppContainer. You can start any process in AppContainer - it is not specific to Windows Store.

Reply:
Notice that the function name is right already: IsAppContainerProcess

Reply:
Also, you can delete all mentions of WP8